optimize/docker-compose.ccsm-without-optimize.yml

```diff
@@ -95,7 +96,8 @@ services:
       - discovery.type=single-node
       - xpack.security.enabled=false
       - action.destructive_requires_name=false
-      - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+      - ES_JAVA_OPTS=-Xms512m -Xmx512m -XX:UseSVE=0
```

Comment:
Just one last thing that I missed last time around, sorry about that. We are reducing the memory allocation here. Please leave it as 1g.